modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/liferay-clay.tld

```diff
@@ -1817,6 +1817,12 @@
 			<required>false</required>
 			<rtexprvalue>true</rtexprvalue>
 		</attribute>
+		<attribute>
+			<description></description>
+			<name>showDesignImprovements</name>
+			<required>false</required>
+			<rtexprvalue>true</rtexprvalue>
+		</attribute>
```

Comment:
@tinycarol  Do we really need this? It is not used right now, and adding it here would mean that if we want to remove it, it will be a breaking change. I'd remove this and rely only on the configuration entry, what do you think?

Reply:
Yep, nice catch, we don't need to add an attribute to the tag, just pass the value to the React component props

Reply:
@victorg1991 Makes sense, on it! Thanks

Reply:
Actually I'm surprised that SF doesn't complain about having an attribute defined in the `tld` but no setter in the Java Class